src/software/simulation/simulator_robot.cpp

```diff
@@ -165,7 +166,7 @@ void SimulatorRobot::kick(float speed_m_per_s)
                 robot_orientation_vector.normalize(ball_head_on_momentum.length()));
             ball->applyImpulse(kick_impulse);
 
-            dribbler_ball.can_be_chicked = false;
+            dribbler_ball.can_be_controlled = false;
```

Comment:
In the case where autokick and dribbling are on, are we guaranteed to get to here before the dribbler checks for the flag?

Reply:
Yes. The "start contact" callbacks are called before the "pre-solve" callbacks (which is bound to applying dribbling force).

So in that case:
1. `onDribblerBallStartContactCallback` is called
2. damping force is applied
3. autokick is triggered if it's on
4. the `onDribblerBallContact` callback is called
4. Dribbling force would be applied here if the ball was not kicked

Reply:
Is this documented somewhere?

Reply:
added documentation now.